locale/en_US/manager.po

```diff
@@ -903,6 +906,12 @@ msgstr "These genres are used for file-naming purposes and are presented in a pu
 msgid "manager.setup.reviewProcessEmailDescription"
 msgstr "Editors send\tReviewers the request to review with the submission attached to the email. Reviewers email editors their assent (or regrets), as well as the review and recommendation. Editors enter Reviewers' assent (or regrets), as well as the review and recommendation on the submission's Review page, to record the review process."
 
+msgid "manager.setup.disableSubmissions.notAccepting"
+msgstr "This press is not accepting submissions at this time. Visit the <i>workflow settings</i> to allow submissions."	
+
+msgid "manager.setup.disableSubmissions.description"
+msgstr "Prevent users from submitting new articles to the press. Submissions can be disabled for individual press series on the <a href="{$url}">press series</a> settings page."
```

Comment:
The URL for this has an anchor of `#section`. I think that's fine, but can you change the tab id from "series" to "section" in `templates/management/context.tpl`? That will get the link working and align the two applications.